src/utility.js

```diff
@@ -53,14 +53,15 @@ function parseUncssrc(filename) {
 function parsePaths(source, stylesheets, options) {
     return stylesheets.map(function (sheet) {
         var sourceProtocol;
+        var isLocalFile = sheet.substr(0, 5) === 'file:';
 
         if (sheet.substr(0, 4) === 'http') {
```

Comment:
Perhaps this should be extended to handle http and https protocol URLs, while you're in here messing with things?